src/components/Organisms/AssetTile/stories/AssetTileGridItemStory.tsx

```diff
@@ -55,7 +56,7 @@ export const AssetTileGridItemStory = () => {
 
   const selectedFeatures = (options('Features', featureOptions, Object.values(featureOptions), {
     display: 'check',
-  }) as unknown) as string[]
+  }) as OptionsTypeKnobValue) as string[]
```

Comment:
Can just remove cast types as it's only a story
```suggestion
  }))
```